applications/management/commands/assign_application_area_type.py

```diff
@@ -0,0 +1,32 @@
+from datetime import date
+
+from django.core.management import BaseCommand
+
+from applications.enums import ApplicationAreaType
+from applications.models import WinterStorageApplication
+
+
+class Command(BaseCommand):
+    help = "Assign Application area_type for existing applications"
+
+    def handle(self, *args, **options):
+        # Get ws applictions created after 14.9.2020
+        new_applications = WinterStorageApplication.objects.filter(
+            created_at__gt=date(2020, 9, 14)
+        )
+
+        for application in new_applications:
+            application.area_type = application.resolve_area_type()
+            application.save()
+
+        # Get ws applictions created before 14.9.2020
+        old_applications = WinterStorageApplication.objects.filter(
+            created_at__lt=date(2020, 9, 14)
```

Comment:
we probably need to use `__lte`, since otherwise applications left on 14.9 will be missed.

Reply:
the expressions seem to produce following SQLs:
`created_at__gt=date(2020, 9, 14) -> WHERE "applications_winterstorageapplication"."created_at" > '2020-09-14T00:00:00+03:00'`
`created_at__lt=date(2020, 9, 14) -> WHERE "applications_winterstorageapplication"."created_at" < '2020-09-14T00:00:00+03:00'`
so I think we are ok?

Reply:
well it misses the application(s) that are created exactly at `2020-09-14T00:00:00+03:00` :D I'll add `lte` to second of so that it's bullet proof

Reply:
hmmm, seems like `__gt` is essentially `__gte`, while `__lt` is as expected `__lt`. maybe then your current solution is ok? can you double check that it catches the applications ON the day we use for these lookups?

Reply:
I guess there's no harm in adding the `__lte` 🤷

Reply:
agree what @a-rmz said and dubbel checked that it does catch the applications that are created on that day (2020-09-14). So I think we are good to go.